packages/vue/src/components/organisms/SfAccordion/SfAccordion.js

```diff
@@ -4,5 +4,66 @@ import SfAccordionItem from "./_internal/SfAccordionItem.vue";
 Vue.component("SfAccordionItem", SfAccordionItem);
 
 export default {
-  name: "SfAccordion"
+  name: "SfAccordion",
+  props: {
+    /**
+     * Array of items to populate the default markup
+     */
+    items: {
+      type: Array,
+      default: []
+    },
+    /**
+     * Callback function to handle click on accordion item content element
+     */
+    contentClickedCallback: {
+      type: Function,
+      default: () => {}
+    },
+    /**
+     * Allows to open multiple accordion items if set to "true"
+     */
+    multiple: {
+      type: Boolean,
+      default: false
+    },
+    /**
+     * Opens the first accordion item if set to "true"
+     */
+    firstOpen: {
+      type: Boolean,
+      default: false
+    },
+    /**
+     * Allows to use two different markups for opened and closed header
+     */
+    openHeaderStyle: {
+      type: Boolean,
+      default: false
+    }
+  },
+  methods: {
+    toggle(slotId) {
+      if (!this.multiple) {
+        this.$children.forEach(child => {
+          child._uid === slotId
+            ? (child.isOpen = !child.isOpen)
+            : (child.isOpen = false);
+        });
+      } else {
+        const clickedHeader = this.$children.find(child => {
+          return child._uid === slotId;
+        });
+        clickedHeader.isOpen = !clickedHeader.isOpen;
+      }
+    },
+    openFirst() {
+      this.firstOpen ? (this.$children[0].isOpen = true) : "";
```

Comment:
You don't have to use **conditional operator** if you just want to perform some operation, based on some condition. Use `&&` operator instead : `this.firstOpen && (this.$children[0].isOpen = true)`, and you don't have to put this dummy empty string value in negative option.

However for me, still more readable is old fashioned `if`.

Reply:
Hi @dimitor115
In general I agree, but here the easiest way is to just simply:
<code>this.$children[0].isOpen = this.firstOpen</code>. But probably without your comment I would have never refactored this line ;-)